### ISSUE TITLE
missing leases permission in chart 0.8.0

### ISSUE THREAD

**Issue by travisghansen** [isindir/sops-secrets-operator#69]
```
E0514 21:55:17.927234       1 leaderelection.go:325] error initially creating leader election record: leases.coordination.k8s.io is forbidden: User "system:serviceaccount:operators:sops-operator-sops-secrets-operator" cannot create resource "leases" in API group "coordination.k8s.io" in the namespace "operators"
```

**Comment by isindir** [comment]
@travisghansen which version of k8s are you deploying on? I thought I added that permission..

**Comment by travisghansen** [comment]
This is on 1.19...I added create manually for now and all is well but argocd complains about configuration drift.

**Comment by isindir** [comment]
Fixed by #70 and Confirmed in #65, Closing this issue
